src/bin/vip-import-sql.js

```diff
@@ -351,7 +350,7 @@ const displayPlaybook = ( {
 				);
 			}
 			console.log( chalk.yellow( 'The following sites will be affected by the import:' ) );
-			multiSiteBreakdown.map( siteObject => {
+			multiSiteBreakdown.forEach( siteObject => {
```

Comment:
When the callback passed to `map` does not return anything, `map` should be replaced with `forEach`.